admin/server/organizations.go

```diff
@@ -186,6 +186,33 @@ func (s *Server) UpdateOrganization(ctx context.Context, req *adminv1.UpdateOrga
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	// Iterate over organization's projects, and push updated annotations to deployments
+	afterProjectName := ""
+	for {
+		projects, err := s.admin.DB.FindProjectsForOrganization(ctx, org.ID, afterProjectName, 10)
+		if err != nil {
+			return nil, err
+		}
+
+		if len(projects) == 0 {
+			break
+		}
+
+		for _, project := range projects {
+			err := s.admin.UpdateProjectAnnotations(ctx, project, map[string]string{
+				"organization_id":   org.ID,
+				"organization_name": org.Name,
+				"project_id":        project.ID,
+				"project_name":      project.Name,
+			})
```

Comment:
We should only have one place anywhere that we construct this payload, so it doesn't go out of sync (right now, it's also created in `createDeployment`)